src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "../Helpers/StringConverter.h"
 
+#include "..//Globals/Settings.h"
```

Comment:
Double slash used, typo?

Reply:
yep, sorry.. will correct it.. (too. much http url typing I guess ;)

thanks!